### PR TITLE
add localnet

### LINKED ISSUE #3: add localnet setting - signle
like `finschia-js/script/finschia`

### PR DESCRIPTION
**Pull request by loin3**
Resolves #3 

All localnet settings are from finschia-js/scripts/finschia.
From finschia-s/scripts/finschia contract settings are deleted and finschia version changed to `1.0.0-dynamiclink1`
Description about 11 accounts (9 ordinary accounts, 1 multisig, 1 validator) is added to README.

### CODE REVIEW

File: localnet/finschia/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Local simd development network
```
Comment (zemyblue) [inline localnet/finschia/README.md]: `simd` -> `finschia single node`

File: localnet/finschia/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Starting the blockchain
+
+Run the following:
+
+```shell
+cd scripts/finschia
```
Comment (zemyblue) [inline localnet/finschia/README.md]: How about removing this line or changing appropriate directory?

File: localnet/finschia/template/run_finschia.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+cp -R "/template/.finschia" /root/.finschia
+mkdir -p /root/log
+fnsad start --rpc.laddr tcp://0.0.0.0:26658 --trace
```
Comment (zemyblue) [inline localnet/finschia/template/run_finschia.sh]: `26658` -> `26657`

File: localnet/finschia/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Local finschia single node development network
+
```
Comment (zemyblue) [inline localnet/finschia/README.md]: Please add a requirements. 
- docker

**Review by zemyblue — COMMENTED** [review]
This scripts does not work. Please check the following errors.

```
Unable to find image 'finschia/finschianode:1.0.0-dynamiclink1' locally
1.0.0-dynamiclink1: Pulling from finschia/finschianode
edb6bdbacee9: Already exists
4f4fb700ef54: Already exists
7aab9fa447c3: Pulling fs layer
7aab9fa447c3: Verifying Checksum
7aab9fa447c3: Download complete
7aab9fa447c3: Pull complete
Digest: sha256:34fe4d5d274c9a7df362a0e8a408e6ae16daeeffa12c50201c122f023eba1339
Status: Downloaded newer image for finschia/finschianode:1.0.0-dynamiclink1
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:1317 -> 0.0.0.0:0: listen tcp 0.0.0.0:1317: bind: address already in use.
time="2023-06-28T15:23:16+09:00" level=error msg="error waiting for container: context canceled"
```

**Comment by zemyblue** [comment]
> This scripts does not work. Please check the following errors.
> 
> ```
> Unable to find image 'finschia/finschianode:1.0.0-dynamiclink1' locally
> 1.0.0-dynamiclink1: Pulling from finschia/finschianode
> edb6bdbacee9: Already exists
> 4f4fb700ef54: Already exists
> 7aab9fa447c3: Pulling fs layer
> 7aab9fa447c3: Verifying Checksum
> 7aab9fa447c3: Download complete
> 7aab9fa447c3: Pull complete
> Digest: sha256:34fe4d5d274c9a7df362a0e8a408e6ae16daeeffa12c50201c122f023eba1339
> Status: Downloaded newer image for finschia/finschianode:1.0.0-dynamiclink1
> docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:1317 -> 0.0.0.0:0: listen tcp 0.0.0.0:1317: bind: address already in use.
> time="2023-06-28T15:23:16+09:00" level=error msg="error waiting for container: context canceled"
> ```

This is my local port problem. Now it works well.

**Review by zemyblue — DISMISSED** [review]

**Review by tkxkd0159 — DISMISSED** [review]

**Comment by tkxkd0159** [comment]
I did some chores

**Review by zemyblue — DISMISSED** [review]

**Review by tkxkd0159 — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**